ui/widgets/calendar.js

```diff
@@ -150,7 +151,14 @@ return $.widget( "ui.calendar", {
 		}
 
 		if ( this._needsRefresh() ) {
-			this._refresh();
+			if ( this.options.numberOfMonths > 1 && this.date.year() === this.viewDate.year() ) {
+				this.viewDate.adjust( "M", this.options.numberOfMonths *
+					( this.date.month() > this.viewDate.month() ? 1 : -1 )
+				);
+				this.refresh();
+			} else {
+				this._refresh();
```

Comment:
Based on usage here and method naming, I can't tell the difference between `_refresh()` and `refresh()`. Maybe rename `_refresh()` and make callers call both that and `refresh()`?

Btw. the multi line comment in front of `refresh()` doesn't help either, it mostly sounds like a "avoid calling this" warning.